alacritty/src/display/window.rs

```diff
@@ -63,7 +61,7 @@ static WINDOW_ICON: &[u8] = include_bytes!("../../extra/logo/compat/alacritty-te
 
 /// This should match the definition of IDI_ICON from `alacritty.rc`.
 #[cfg(windows)]
-const IDI_ICON: WORD = 0x101;
+const IDI_ICON: u16 = 0x101;
```

Comment:
I guess windows-sys don't have aliases for those?

Reply:
That's right! I didn't find this in the documentation, the closest is PWSTR which is *const u16.